core/http/server-spring/src/main/java/org/eclipse/rdf4j/http/server/repository/transaction/Transaction.java

```diff
@@ -57,7 +60,7 @@
  * 
  * @author Jeen Broekstra
  */
-class Transaction {
+class Transaction implements AutoCloseable {
```

Comment:
This shouldn't be necessary. The `ActiveTransactionRegistry` is responsible for closing the `Transaction` object when done.